### ISSUE TITLE
html file is produced with strange name (content hash etc) when it is placed inside src folder

### ISSUE THREAD

**Issue by denis1stomin** [parcel-bundler/parcel#1500]
Hey guys,
Searched by "hashed" etc..

# 🐛 bug report

I have a folder somewhere inside my repo some/path/webui.
This folder contains standard package structure and src folder in it.
I moved index,html file into src folder since for me it is a source code.
(I'm a pro developer but really new in those all frontend things :))
Now when I run "npx parcel build" (production mode) or globally "parcel build" it produces a file like
dist/src.4320c2f6.html. Looks like it treats all src/* files like "aren't entry points".
For me it looks like a bug.
So the questions are:
1. Is it a bug for you and can it be easily fixed in future versions?
2. If it is "by design" (and I'm doing something wrong), how you'd describe (your vision) where html files should be places and why?

## 🎛 Configuration (.babelrc, package.json, cli command)

https://github.com/denis1stomin/OKRPortal/blob/ui-ci/webui/.babelrc
https://github.com/denis1stomin/OKRPortal/blob/ui-ci/webui/package.json

## 🤔 Expected Behavior

I'd like to see dist/index.html as mentioned here
https://github.com/parcel-bundler/parcel/pull/1025
I guess I use prod mode.

## 😯 Current Behavior

I've got dist/src.4320c2f6.html

## 💁 Possible Solution

dunno :)

## 🔦 Context

Just simple app to learn modern stuff

## 💻 Code Sample

https://github.com/denis1stomin/OKRPortal/tree/ui-ci/webui

## 🌍 Your Environment

Ubuntu 17

**Comment by DeMoorJasper** [comment]
Are you using parcel 1.8.1 (local or global, please check both)

**Comment by denis1stomin** [comment]
Both produce the same result

**Comment by DeMoorJasper** [comment]
Also did you define an entrypoint like `parcel build index.html` would be usefull to know your config

**Comment by KeineLimonade** [comment]
You did not specify an entrypoint. Replace your npm.start script with "parcel build src/index.html"

**Comment by denis1stomin** [comment]
@KeineLimonade @DeMoorJasper thank you!
I've specified an entrypoint as src/index.html and I've seen an error again
"Unexpected identifier
    async start() {"
which I saw before and decided that I'm doing something wrong.. :)
Then I found that nodejs v8 is a minimal version to use with parcel.
So I've updated my v6* to v10 and now it works as expected :)
Thanks for your help again!

So what is this mode for "parcel build" without an entrypoint?
May be it is not bad idea to check current node version and warn an user?
